packages/@okta/courage-for-signin-widget/webpack.config.js

```diff
@@ -6,10 +6,12 @@ const CopyWebpackPlugin = require('copy-webpack-plugin');
 const PACKAGE_JSON = require('./package.json');
 
 const EMPTY = resolve(__dirname, 'src/empty');
-const SHARED_JS = resolve(__dirname, 'node_modules/@okta/courage/src');
-const COURAGE_DIST = resolve(__dirname, 'node_modules/@okta/courage/dist');
-const PUBLISH_DIR = resolve(__dirname, '../courage-dist');
-const I18N_DIR = resolve(__dirname, '../i18n');
+const NODE_MODULES_SRC = resolve(__dirname, 'node_modules/@okta');
+const NODE_MODULES_DEST = resolve(__dirname, '..');
```

Comment:
super-nit: Switch the order of these vars to gradually increase:

```suggestion
const NODE_MODULES = resolve(__dirname, '..');
const OKTA_NODE_MODULES = resolve(NODE_MODULES_DEST, '@okta');
```